hugo/assets/scss/components/anchor.scss

```diff
@@ -0,0 +1,27 @@
+@import '../mixins/screen';
+
+.anchor {
+    $self: &;
+
+    left: 4px;
```

Comment:
Using left leaves a gap between h and anchor and when you hover in the gap the anchor is not visible. Maybe solve with padding-right instead?